css/main.css

```
@@ -100,15 +100,15 @@ div#nav li a{
 
 /* -Contenedores de Secciones principales -Contenedores de Secciones principales -Contenedores de Secciones principales -Contenedores de Secciones principales*/
 
-div#divHola h1{
+div#div-hola h1 {
     color: #00ADBB;
     font-size: 80px;
 }
 
-div#divHola{
+div#div-hola {
     text-align: center;
     display: block;
-    background-color:#ffffff;
+    background-color: #ffffff;
     padding-bottom: 45px;
```

Can we use the shorthand in here 
```suggestion
    padding: 0 45px 45px 45px;
```

---

done with margin and padding